server/apps/distributed-app/docs/modules/ROOT/pages/operate/index.adoc

```diff
@@ -24,6 +24,8 @@ distributed James operators.
 
 We also included a guide for xref:operate/migrating.adoc[migrating existing data] into the distributed server.
 
+We introduced xref:operate/db-benchmark.adoc[James database's base performance and how to benchmark] them.
```

Comment:
It would be cool to also document (or point to) tools to benchmark james itself ;-) (other issue)

Reply:
created: https://github.com/linagora/james-project/issues/4504